### PR TITLE
Add 'or mounted resinOS image'

### PR DESCRIPTION
**Pull request by MoranF**
Connects-To: #764

Change-Type: patch
cqqq
---- Autogenerated Waffleboard Connection: Connects to #764

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
     "resin-image-fs": "^2.3.0",
     "resin-image-manager": "^5.0.0",
     "resin-preload": "^5.0.0",
-    "resin-sdk": "^7.0.0",
+    "resin-sdk": "^8.1.0",
```
Comment (pimterry) [inline package.json]: This is a surprising change to include here! Why is this in this PR? Does this definitely not need other changes anywhere?

Comment (thgreasi) [reply package.json]: It contains the new getDashboardUrl for sure.

Comment (MoranF) [reply package.json]: @pimterry , if I didn't change the resin-sdk version I had errors in the build, but you are right we need a new pull request for that, checking all the changes in the new sdk version. I will try to check which 7.x.x version I should use

**Comment by MoranF** [comment]
@lekkas , @zwalchuk , can you approve this PR please?

**Pull request merged**